tests/test_config.py

```diff
@@ -25,6 +26,30 @@
 from gpflow.utilities import to_default_float, to_default_int
 
 
+def test_env_variables():
+    os.environ["GPFLOW_INT"] = "int64"
+    os.environ["GPFLOW_FLOAT"] = "float32"
+    os.environ["GPFLOW_POSITIVE_BIJECTOR"] = "exp"
+    os.environ["GPFLOW_POSITIVE_MINIMUM"] = "1e-3"
+    os.environ["GPFLOW_SUMMARY_FMT"] = "simple"
+    os.environ["GPFLOW_JITTER"] = "1e-2"
+
+    config = gpflow.config.Config()
+    assert config.int == np.int64
+    assert config.float == np.float32
+    assert config.positive_bijector == "exp"
+    assert config.positive_minimum == 1e-3
+    assert config.summary_fmt == "simple"
+    assert config.jitter == 1e-2
+
+
+@pytest.mark.parametrize("key", ["INT", "FLOAT", "POSITIVE_BIJECTOR", "POSITIVE_MINIMUM", "JITTER"])
```

Comment:
Could we test this programmatically - e.g. taking Config.__dict__.keys() or something like that, and manually exclude summary_fmt - to make sure new cases will be tested if at some point someone adds a new field to the config ?

Reply:
done